raco/language/myrialang.py

```diff
@@ -806,6 +838,9 @@ def __repr__(self):
     def num_tuples(self):
         return self._num_tuples
 
+    def partitioning(self):
```

Comment:
If, say, a `MyriaScan` (`TableScan` in MyriaX) is optimized to a `MyriaQueryScan` by the `PushIntoSQL` rule, don't we need to preserve the partitioning info from `MyriaScan` for any consumers of the new `MyriaQueryScan`? If so, I guess its constructor should take a `partitioning` arg. Or maybe I'm missing something and all the partitioning info is propagated up the query tree and relevant optimizations are performed before any expressions are optimized into `MyriaQueryScan`.